lib/infrastructure/utils.js

```diff
@@ -36,22 +37,11 @@ module.exports = function Utils() {
     }
 
     function _removeDirectory(dirPath) {
-        if(!fs.existsSync(dirPath)) {
-            return;
-        }
-
-        fs.readdirSync(dirPath).forEach(function(file) {
-            var currentFilePathPath = path.join(dirPath, file);
-
-            if(fs.lstatSync(currentFilePathPath).isDirectory()) {
-                _removeDirectory(currentFilePathPath);
-            }
-            else {
-                fs.unlinkSync(currentFilePathPath);
+        rmdir(dirPath, function(error) {
```

Comment:
this function is called async. ```rmdir.sync``` should be used here or every other function using this should be rewritten to be async.